### DISCUSSION TITLE
Gehäuse für Pylontech Speicher

### DISCUSSION THREAD

**Discussion by Phantomias2006** [hoylabs/OpenDTU-OnBattery#520] [General]
Hi,

die letzten Wochen war ich damit beschäftigt ein Gehäuse für den Speicher und Elektronik zu Zeichnen, Fräsen und bauen.
Es können theoretisch 2x Pylontech US 3000C Speicher verbaut werden. Ich selber habe nur einen verbaut und werde den unteren Einschub für Notstrom verwenden (Spannungswandler)
Deshalb habe ich in der Frontblende auch die Buchsen für "NOT" verbaut. Im falle eines Stromausfalls werden die Kabel einfach von NETZ auf NOT umgesteckt und dann ist der Spannungswandler aktiv. 

Anbei ein paar Bilder wie ich es realisiert habe:

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/15437240/072523e4-415d-40a9-8006-4f034b08a0c8)

![IMG_8615](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/15437240/5205266b-11a8-4a25-8674-df27092206d8)
![IMG_8616](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/15437240/4e471706-1df4-4108-9291-f24b2e3a892c)
![IMG_8617](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/15437240/7561b64d-b049-4c6a-ad5e-8ef32f3d5567)

Edit:
Auf Wunsch werde ich hier nach und nach die Files und Links der verwendeten Teile ergänzen. Bei Fragen einfach melden.

### **Stückliste (nicht Vollständig!)**

**Amazon:**

- [Pylontech US 3000C](https://www.amazon.de/gp/product/B08S6T63Q4/?tag=wlanthermo-21)
- [Hoymiles HM-600](https://www.amazon.de/gp/product/B08KR76QG5/?tag=wlanthermo-21)
- [ESP32 Breakout Board](https://www.amazon.de/dp/B0B7L6617P/?tag=wlanthermo-21)
- [MEAN WELL Netzteil 5V](https://www.amazon.de/dp/B06XWQSJGW/?tag=wlanthermo-21)
- [MEAN WELL Netzteil 12V](https://www.amazon.de/dp/B06XWSYRCF/?tag=wlanthermo-21)
- [Batteriekabel](https://www.amazon.de/dp/B09BQD1M1S/?tag=wlanthermo-21)
- [Möbelrollen](https://www.amazon.de/dp/B0B83T6G8P/?tag=wlanthermo-21)
- [5V Relais](https://www.amazon.de/dp/B07VYX5GK4/?tag=wlanthermo-21)
- [Waveshare SN65HVD230 CAN Board](https://www.amazon.de/SN65HVD230-Board-Connecting-Communication-Development/dp/B00KM6XMXO/ref=sr_1_3?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=VSBVOVM6FTUX&keywords=Waveshare%2BSN65HVD230&qid=1698392048&sprefix=waveshare%2Bsn65hvd230%2B%2Caps%2C126&sr=8-3&th=1)
- NodeMCU?
- MCP2515 CAN Modul
- [2x 120mm Gehäuselüfter 12V](https://www.amazon.de/dp/B01L6QQJ88/?tag=wlanthermo-21)
- [1x 120mm Gehäuselüfter Staubabdeckung](https://www.amazon.de/dp/B0989BMYV8/?tag=wlanthermo-21)

**Reichel:**

- [1x LAN Einbaubuchse](https://www.reichelt.de/at/de/rj45-adapter-frontplattenmontage-cat-5e-ft-serie-cliff-cp30220x-p227537.html?&trstct=pos_0&nbc=1)
- [2x MC4 Kupplungsbuchse](https://www.reichelt.de/at/de/mc4-evo2-kupplungsbuchse-mc4-pv-kbt4-evo-p337269.html?&trstct=pos_0&nbc=1)
- [2x MC4 Kupplungsstecker](https://www.reichelt.de/at/de/mc4-evo2-kupplungsstecker-mc4-pv-kst4-evo-p337270.html?&trstct=pos_2&nbc=1)
- [1x WAGO Befestigungsadapter Blau](https://www.reichelt.at/at/de/befestigungsadapter-fuer-2-3-5-leiter-verbindungkl-4-mm--wago-221-506-p310154.html?r=1)
- [1x WAGO Befestigungsadapter Dunkelgrau-Gelb](https://www.reichelt.at/at/de/befestigungsadapter-fuer-2-3-5-leiter-verbindungkl-4mm--wago-221-5053-p310155.html?&trstct=vrt_pdn&nbc=1)
- [1x WAGO Befestigungsadapter Orange](https://www.reichelt.at/at/de/befestigungsadapter-221-fuer-2-3-5-leiter-verbindungskl-ora-wago-221-500-p182323.html?&nbc=1&trstct=lsbght_sldr::310155)
- [1x Verdrahtungskanal 37.5x50 mm](https://www.reichelt.at/at/de/verdrahtungskanal-37-5x50-mm-1000-mm-din37-5-50-p338244.html?&trstct=pol_2&nbc=1)
- [1x Verdrahtungskanal 25x25 mm](https://www.reichelt.at/at/de/verdrahtungskanal-25x25-mm-1000-mm-din25-25-p338241.html?&trstct=pol_6&nbc=1)
- [1x BETTERI BC01F Buchse](https://www.reichelt.at/at/de/betteri-bc01-buchse-female-ac-3-polig-ip67-betteri-bc01f-p331499.html?&trstct=pos_0&nbc=1)
- [1x Wieland Steckverbinder RST20I3S S1 YR1 V SW](https://www.reichelt.de/at/de/steckerteil-mit-kabelverschraubung-6-10mm-schraubanschluss-rst20i3-ks2-90-p333667.html?&trstct=pos_5&nbc=1)
- [1x Wieland RST20 PV DOSE AP](https://www.reichelt.de/at/de/index.html?ACTION=446&LA=446&nbc=1&q=rst20%20pv%20dose%20ap)
- [1x USB-C Stromkabel](https://www.reichelt.de/at/de/usb-c-stecker-auf-freie-enden-sw-50-cm-usb-c-awg22-50-p292506.html?&trstct=pos_1&nbc=1)

**Distrelect:**

- [1x Ringkabelschuhe 6,4mm 16mm2](https://www.distrelec.at/de/ringkabelschuh-nicht-isoliert-4mm-16mm-kupfer-packung-10-stueck-rnd-connect-rnd-465-01076/p/30176831?trackQuery=kabelschuhe&pos=133&origPos=33&page=3&pageSize=50&origPageSize=50&track=true)
- [1-2x Amphenol SLPRATPSB Schwarz](https://www.distrelec.at/de/7mm-anschluss-stecker-schwarz-120a-pole-amphenol-industrial-slpratpsb/p/30215321)
- [1-2x Amphenol SLPRATPSR ROT](https://www.distrelec.at/de/7mm-anschluss-stecker-rot-120a-pole-amphenol-industrial-slpratpsr/p/30215323)
- [1x Hutschiene](https://www.distrelec.at/de/hutschiene-35x7-5mm-stahl-250mm-phoenix-contact-1207639/p/14808915?trackQuery=hutschiene&pos=5&origPos=5&origPageSize=50&track=true)

**Andere Shops:**

- Huawei R4850G2
- Anschlusskabel Huawei R4850G2

### Das Gehäuse als STEP Datei:

Ich denke mit der STEP kann man alles ableiten für einen Nachbau. Das Gehäuse besteht aus 12mm Siebdruckplatten.
Im Bereich der Elektronik habe ich noch 5mm Pertinax für Hitze/Brandschutz verwendet! Diese sind in der STEP aktuell nicht enthalten.
[Pylontech assembling v1.zip](https://github.com/helgeerbe/OpenDTU-OnBattery/files/13187115/Pylontech.assembling.v1.zip)

**Comment by swingstate** [comment]
Passen die Temperaturen wenn der HM auf Volllast läuft?

**Comment by Phantomias2006** [comment]
Kann ich noch nicht sagen da aktuell der Deckel noch nicht geschlossen ist, aber es sind 2x 120mm 12V Lüfter verbaut (1x rein, 1x raus). Ich denke es wird keine Thermischen Probleme geben

**Reply by swingstate** [reply]
Ich würde vielleicht versuchen in die Mitte ein Luftleitblech / Holz zu installieren. Damit die Luft auch zirkulieren muss. Ansonsten schön gebaut. Mein HM 1500 hängt draußen und wird unter Last gerne mal >60° heiss.

**Reply by swingstate** [reply]
welches CAN Modul hast Du installiert und musstest Du daran etwas ändern, damit die Pylontech erkannt wird?

**Reply by Phantomias2006** [reply]
https://www.amazon.de/SN65HVD230-Board-Connecting-Communication-Development/dp/B00KM6XMXO/ref=mp_s_a_1_3?crid=1K3A1TK78N03Y&keywords=waveshare%2Bsn65hvd230%2Bcan&qid=1689077866&sprefix=waveshare%2Bsn%2Caps%2C169&sr=8-3&th=1
Ging out of the Box. Einfach angeschlossen, Pins konfiguriert, aktiviert und alles wurde korrekt ausgelesen.

**Comment by alexseuf** [comment]
Tolle Realisierung, gibt es evtl auch ausschnittsweise Dokumente wie Stromlaufplan, Masszeichnungen, Stücklisten? Das würde sicherlich vielen den Nachbau dieser sehr durchdaten Lösung vereinfachen.

**Reply by Phantomias2006** [reply]
Aktuell noch nicht, aber Gehäuse Zeichnungen kann ich gerne veröffentlichen. Dauert aber aber noch etwas, muss noch ein paar Kleinigkeiten in den Zeichnungen korrigieren…

**Reply by Phantomias2006** [reply]
@alexseuf 
Ich habe im ersten Post begonnen die Stückliste aufzuschreiben. LG

**Reply by Phantomias2006** [reply]
@alexseuf ich habe das Projekt als STEP exportiert und im ersten Post ergänzt. Ich denke damit kann man alles für einen Nachbau ableiten.

**Comment by Manos1966** [comment]
Respekt! 🥇

**Reply by Manos1966** [reply]
Eine frage @Phantomias2006 
Wo ist das NRF24 Sendmodul? (ich kann es im Bild nicht erkennen)

**Reply by Phantomias2006** [reply]
Hi, das hast du gut beobachtet. Aktuell ist kein NRF Modul verbaut. Aufgrund der damaligen Stabilitätsprobleme hatte ich mich entschieden eine openDTU mit NRF extern zu verwenden und alles mittels NodeRed zu steuern. Hier starte ich auch die DTU automatisch neu falls es Verbindungsprobleme etc. gibt. Aktuell läuft die Software aber sehr stabil und sollte dieser [PR](https://github.com/helgeerbe/OpenDTU-OnBattery/pull/480) es ins Release schaffen werde ich vermutlich einen NRF verbauen.

**Comment by Manos1966** [comment]
@Phantomias2006 Danke fuer die Ausfuehrliche Erkalerung und die Plaene, die du gepostet hast 👍 

Ich hoffe du hast nichts dagegen, wenn wir deine Erzaehlungen in die Wiki addieren
unter [Builds & Examples](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Builds-%26-Examples)

**Reply by Phantomias2006** [reply]
@Manos1966 ihr könnt es gerne verlinken. Es wäre schade wenn die ganze Arbeit nur für das eine Gehäuse gewesen wäre. Wenn euch noch Unterlagen fehlen dann einfach Bescheid geben, sofern möglich werde ich diese ergänzen. LG

**Comment by Head** [comment]
Sehr cooles Projekt. Meinen Respekt für den Aufbau!
Was ich nicht ganz verstehe: Was macht die Kiste? Was ist das silberne Gerät? Wo ist der Laderegler, wo der AC-Anschluss?
(Ja, stimmt, man könnte die Busbar weglassen und den Laderegler mit an den Pylontech stecken, oder?)

**Reply by Phantomias2006** [reply]
Hi,

ich habe natürlich ganz vergessen mein Setup zu beschreiben...
Ich habe am Dach 4 PV Module welche am HM-1500 hängen. Dieser ist auch am Dach verbaut und ich komme direkt mit 230V ins Haus.
Also habe ich keinen Victron Laderegler verbaut und kann meinen Speicher nur über die 230V laden. Dafür habe ich ein Huawei R4850G2 Netzteil mit CAN Anschluss gekauft. Dieser lädt den Speicher mit dem Überschuss welcher sonst ins Netz ging.
Außerdem ist ein HM-600 Wechselrichter verbaut welcher ins Netz bei Bedarf einspeist.

Ich hoffe es war verständlich, ansonsten kann ich es gerne noch einmal genauer beschreiben.

LG
